### PR TITLE
media-sound/lollypop: add 1.4.35

### PR DESCRIPTION
**Pull request by Nielzsen**
Updated to 1.4.35, added gst-plugins-pulse as a RDEPEND since Lollypop doesn't seem to output anything without it.
Also added a reminder to install the necessary gst-plugins, and updated the source URL.
Tested for a few weeks on my install without issue.

Signed-off-by: Roman Brasseur <nielzsen@protonmail.com>

### CODE REVIEW

File: media-sound/lollypop/lollypop-1.4.35.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
```
Comment (ceamac) [inline media-sound/lollypop/lollypop-1.4.35.ebuild:6]: Please add python 3.11

File: media-sound/lollypop/lollypop-1.4.35.ebuild [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	virtual/pkgconfig
+	test? (
+		dev-libs/appstream-glib
+		dev-util/desktop-file-utils
+	)
+"
+
+RDEPEND="
+	${COMMON_DEPEND}
+        media-plugins/gst-plugins-pulse
```
Comment (ceamac) [inline media-sound/lollypop/lollypop-1.4.35.ebuild:46]: Please use 1 tab instead of 8 spaces

File: media-sound/lollypop/lollypop-1.4.35.ebuild [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+src_install() {
+	meson_src_install
+	python_optimize
+	python_fix_shebang "${ED}/usr/bin"
+}
+
+pkg_postinst() {
+	xdg_pkg_postinst
+	gnome2_schemas_update
+        elog "Remember to install the necessary gst-plugins packages to read your different audio files, or rely on the gst-plugins-meta and it's USE flags."
```
Comment (ceamac) [inline media-sound/lollypop/lollypop-1.4.35.ebuild:66]: Same here.  Also, `its` instead of `it's` and can you try to split this into two shorter lines?

**Review by ceamac — CHANGES_REQUESTED** [review]

**Comment by Nielzsen** [comment]
Done, if anything needs to be updated/changed let me know, I'll do it as soon as I can.

File: media-sound/lollypop/lollypop-1.4.35.ebuild [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+PYTHON_REQ_USE="sqlite"
+inherit python-single-r1 gnome2-utils meson xdg
+
+DESCRIPTION="Modern music player for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/Lollypop"
+SRC_URI="https://adishatz.org/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 ~arm64"
```
Comment (ceamac) [inline media-sound/lollypop/lollypop-1.4.35.ebuild:16]: Yes, please drop to unstable

`ekeyword ~all lollypop-1.4.35.ebuild`

**Comment by Nielzsen** [comment]
Change merged, thanks again for the heads up!

**Pull request closed without merging**